### DISCUSSION TITLE
Going from version 1.18 to 1.19 for Orange Pi 5 Plus

### DISCUSSION THREAD

**Discussion by SilverDesert** [Joshua-Riek/ubuntu-rockchip#235] [General]
Hi,

Going from version 1.18 to 1.19 for Orange Pi 5 Plus, do I need to do a new installation from scratch or can I just do 'sudo apt update && sudo apt upgrade -y' in version 1.18?

Thanks.

**Comment by Joshua-Riek** [comment]
You don't need to do a new install from scratch. All you need to do is a simple update and upgrade with apt.
```
sudo apt-get update
sudo apt-get upgrade
```

**Reply by SilverDesert** [reply]
Thanks
